File: FSWO102-JV/lesson08/handsOnL08/Cow.java
package handsOnL08;

public class Cow implements Mammal {

	@Override
	public void speak() {
		System.out.println("Moo!");

	}

	@Override
	public void run() {
		System.out.println("Cows can run at a top speed of 25 mph!");

	}

	@Override
	public void eat() {
		System.out.println("Cows eat grass.");
		
	}
	
	public static void main(String[] args) {
//		Cow bessie = new Cow();
		Mammal bessie = new Cow();
		bessie.speak();
		bessie.run();
		bessie.eat();
	}

}

File: FSWO102-CS/Lesson08/HandsOnL08/HandsOnL08/Classes/Cow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using HandsOnL08.Interfaces;

namespace HandsOnL08.Classes
{
    public class Cow : IMammal
    {
        public void Eat()
        {
            Console.WriteLine("Cows eat grass.");
        }

        public void Run()
        {
            Console.WriteLine("Cows can run at a top speed of 25 mph!");
        }

        public void Speak()
        {
            Console.WriteLine("Moo!");
        }
    }
}
